engine/env/spark/pom.xml

```diff
@@ -136,6 +136,22 @@
 					<groupId>org.eobjects.metamodel-extras</groupId>
 					<artifactId>MetaModel-extras-access</artifactId>
 				</exclusion>
+                <exclusion>
```

Comment:
I think actually we have had the same concern with regard to stuff like the H2StorageProvider or other implementations that was optional. I guess those runtime errors would only occur if the conf.xml file being read actually used those objects. What I mean is that in many cases you would not want remote components in the conf.xml file anyway, and then I guess it would also not try to class load them because the code is never activated, or?